src/run_part.F90

```diff
@@ -549,15 +579,17 @@ subroutine spec_file_read_run_part(file, run_part_opt, aero_data, &
 
     if (.not. do_restart) then
        env_state_init%elapsed_time = 0d0
-       if (.not. run_part_opt%do_camp_chem) then
+       if (run_part_opt%do_camp_chem) then
+#ifdef PMC_USE_CAMP
+          call gas_data_initialize(gas_data, camp_core)
+#endif
+       else if (run_part_opt%do_tchem) then
+            print*, 'do nothing!'
```

Comment:
It isn't temporary as `gas_data` is now set in `TChem_initialize`. Fixed the logic here so that branch that does nothing was removed in https://github.com/compdyn/partmc/pull/198/commits/769ae1866831500ebc8cc17d1ff6d12b4d9c40d3